agent/bench-scripts/pbench-fio

```diff
@@ -538,7 +539,7 @@ function fio_run_job() {
 		wait
 		debug_log "opening port 8765 on firewall on the clients"
 		for client in `echo $clients | sed -e s/,/" "/g`; do
-			ssh $ssh_opts $client "firewall-cmd --add-port=8765/tcp >/dev/null" &
+			ssh $ssh_opts $client "firewall-cmd --add-port=$fio_server_port/tcp >/dev/null" &
```

Comment:
Sorry, but I don't get your point.  I'm not introducing any new behaviour, I mean, I'm just using a var instead of a hard-code value for fio server port. firewall-cmd command was in the code already.

If it were for me, I would completely delete the firewall-cmd in the code, because of the reasons you mention.

Do you want me to get rid of that firewall-cmd section? I would be more than glad to do it.

Reply:
Bummer, totally mis-read your commit.  Let's not remove that behavior in this commit, adding the port variable is fine. I'll file an issue for tracking the removal of the `firewall-cmd` separately.

Reply:
Thanks for pointing out my error in this review.

I have filed issue #1217.